install/apache-guacamole-install.sh

```diff
@@ -51,6 +51,14 @@ chmod -R g+r /opt/apache-guacamole/tomcat9/conf
 chmod g+x /opt/apache-guacamole/tomcat9/conf
 msg_ok "Setup Apache Tomcat"
 
+msg_info "Creating guacd.conf"
+cat <<EOF >/etc/guacamole/guacd.conf
+[server]
+bind_host = 127.0.0.1
+bind_port = 4822
+EOF
+msg_ok "Created guacd.conf"
```

Comment:
LGTM, but dont create a new section, but it under Creating Service (L108) please.

Reply:
The requested modifications have been made, and I conducted extensive testing before committing the changes. I also added a line to explicitly start `guacd` after creating the service, as it would not function otherwise. Without this line, the user would need to take manual action to start the service, or it would only start automatically after a system reboot.